### ISSUE TITLE
Dynamic height issue for content

### ISSUE THREAD

**Issue by akhileshlpatel** [welisonmenezes/wm-gridfolio#1]
When we keep content bigger for one portfolio it is taking max height for that in "wmg-details" and for other portfolio content also it is taking that same height even if content is small for other and there is long space coming for others. Is there any solution for it?

**Comment by tibewww** [comment]
HI Akhileshipatel; I"m having the exact same issue ! Did you find any fix ??

Thanks !!

**Comment by akhileshlpatel** [comment]
yes we fixed that issue before long time. I will check and let you know

**Comment by tibewww** [comment]
Thank you :) Looking forward to see how you solved it :)

**Comment by akhileshlpatel** [comment]
It might be difficult to explain you how exactly I fixed the issue in my
case. but you can check it here http://www.advantages.net/proof/. If you
check the js jquery.wm-gridfolio-1.0.min.js I have made changes in function
"openContent" and "dynamicCSS".

I recalculate the height of the content based on size of window and data.
You can compare the my js with the original js. Hope this will help you!!!

**Comment by sarfraznawaz3** [comment]
can you please share the jquery.wm-gridfolio-1.0.min.js file? Since in website (http://www.advantages.net/proof/) is no longer avaliable. please share
